opslevel/resource_opslevel_team.go

```diff
@@ -78,13 +80,17 @@ func (teamResource *TeamResource) Metadata(ctx context.Context, req resource.Met
 }
 
 func (teamResource *TeamResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	userRoles := []string{"contributor", "manager"}
```

Comment:
Should not be hardcoded because these can vary by account, see https://github.com/OpsLevel/terraform-provider-opslevel/pull/337#discussion_r1597100780